internal/service/lightsail/database.go

```diff
@@ -0,0 +1,485 @@
+package lightsail
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/lightsail"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+	tftags "github.com/hashicorp/terraform-provider-aws/internal/tags"
+	"github.com/hashicorp/terraform-provider-aws/internal/verify"
+)
+
+func ResourceDatabase() *schema.Resource {
+	return &schema.Resource{
+		Create: ResourceDatabaseCreate,
+		Read:   ResourceDatabaseRead,
+		Update: ResourceDatabaseUpdate,
+		Delete: ResourceDatabaseDelete,
+		Importer: &schema.ResourceImporter{
+			State: ResourceDatabaseImport,
+		},
+
+		Schema: map[string]*schema.Schema{
```

Comment:
we'll need to add a customizediff here to support default tags changes e.g. 
```suggestion
                CustomizeDiff: verify.SetTagsDiff,
                
                Schema: map[string]*schema.Schema{
```

Reply:
CustomizeDiff: verify.SetTagsDiff,